next.config.js

```diff
@@ -29,6 +29,14 @@ const nextConfig = {
     // this could be anything, use latest git hash if it exists
     return process.env.GIT_HASH ?? 'vagovprod'
   },
+  webpack(webpackConfig) {
+    return {
+      ...webpackConfig,
+      optimization: {
+        minimize: isProd,
+      },
+    }
+  },
```

Comment:
Not related to the Lovell fix, but useful for getting better stack traces from SSG, which is how we knew this was borked in the first place. Without this, we get stack traces in the minified code during a `yarn export`, which is basically useless.

Since we're throwing up and failing builds when we fail to build pages, we need to know what went wrong. That's where this comes in to help.